### PR TITLE
en: add some introduction in overview.md

### PR DESCRIPTION
**Pull request by Liuxiaozhen12**
### What is changed, added, or deleted? (Required)

add some introduction in overview.md

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/686
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: en/overview.md [hunk 1]
```diff
@@ -64,12 +75,12 @@ Before using the DM tool, note the following restrictions:
 
     - DM reports an error when it encounters an incompatible DDL statement. To solve this error, you need to manually handle it using dmctl, either skipping this DDL statement or replacing it with a specified DDL statement(s). For details, see [Skip or replace abnormal SQL statements](faq.md#how-to-handle-incompatible-ddl-statements).
 
-+ Sharding
++ Sharding and conflict merging
```
Comment (CharLotteiu) [inline en/overview.md:78]:
```suggestion
+ Sharding merge with conflicts
```

File: en/overview.md [hunk 2]
```diff
@@ -8,7 +8,18 @@ aliases: ['/docs/tidb-data-migration/dev/overview/']
 
 # Data Migration Overview
 
-[TiDB Data Migration](https://github.com/pingcap/dm) (DM) is an integrated data migration task management platform, which supports the full data migration and the incremental data replication from MySQL-compatible databases (such as MySQL, MariaDB, and Aurora MySQL) into TiDB. It can help to reduce the operation cost of data migration and simplify the troubleshooting process.
+[TiDB Data Migration](https://github.com/pingcap/dm) (DM) is an integrated data migration task management platform, which supports the full data migration and the incremental data replication from MySQL-compatible databases (such as MySQL, MariaDB, and Aurora MySQL) into TiDB. It can help to reduce the operation cost of data migration and simplify the troubleshooting process. When using DM for data migration, you need to perform the following operations:
+
+- Deploy a DM Cluster
+- Create upstream data source and save data source access information
+- Create (multiple) data migration tasks to migrate data from data sources to TiDB
```
Comment (CharLotteiu) [inline en/overview.md:15]:
```suggestion
- Create data migration tasks to migrate data from data sources to TiDB
```

**Review by CharLotteiu — COMMENTED** [review]
rest LGTM

File: en/overview.md [hunk 3]
```diff
@@ -26,7 +37,7 @@ The [binlog event filtering](key-features.md#binlog-event-filter) feature means
 
 ### Schema and table routing
 
-The [schema and table routing](key-features.md#table-routing) feature means that DM can migrate a certain table of the source database to the specified table in the downstream. For example, you can migrate the data from the table `test`.`sbtest1` in the source database to the table `test`.`sbtest2` in TiDB. This is also a core feature for merging and migrating sharded databases and tables.
+The [schema and table routing](key-features.md#table-routing) feature means that DM can migrate a certain table of the source database to the specified table in the downstream. For example, you can migrate the data and table structure from the table `test`.`sbtest1` in the source database to the table `test`.`sbtest2` in TiDB. This is also a core feature for merging and migrating sharded databases and tables.
```
Comment (lance6716) [inline en/overview.md:40]:
```suggestion
The [schema and table routing](key-features.md#table-routing) feature means that DM can migrate a certain table of the source database to the specified table in the downstream. For example, you can migrate the table structure and data from the table `test`.`sbtest1` in the source database to the table `test`.`sbtest2` in TiDB. This is also a core feature for merging and migrating sharded databases and tables.
```

**Review by lance6716 — COMMENTED** [review]
rest lgtm

**Review by CharLotteiu — APPROVED** [review]

**Review by lance6716 — APPROVED** [review]

**Pull request merged**